src/app/table/table.component.ts

```diff
@@ -129,7 +139,17 @@ export class GpfTableComponent implements OnChanges, AfterViewChecked {
   }
 
   public getVisibleData(): Array<any> {
```

Comment:
The code in this function is identical to the one in the `onWindowResize` function - you should extract it to a single method